fileresolver/fileResolver.go

```diff
@@ -13,4 +13,7 @@ type FileResolver interface {
 
 	// ResolveFileValue converts a value prefixed with "file:" and replaces it with the file contents.
 	ResolveFileValue(value string) ([]byte, error)
+
+	// ResolveMxscValue converts a value prefixed with "mxsc:" and replaces it with the file contents.
```

Comment:
Let's not overcomplicate the FileResolver, it's the same logic, no matter what type of file. The parser can use the same `ResolveFileValue` for both, it's just that there's some additional post-processing for mxsc.